docs/content/docs/testing/testing.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Testing and automation
+menuPosition: 3
+---
+
+## Overview
+
+Testing and automation are crucial to maintaining the quality and longevity of your code.  Internally, Fluid has a range of unit and integration tests powered by Mocha, Jest, and webpack.  Tests that need to run against a service are backed by [Tinylicious]({{< relref tinylicious.md >}}) or a test tenant of a [live service]({{< relref service-options.md >}}) such as [Azure Fluid Relay]({{< relref azure-frs.md >}}).
+
+This document will explain how to use these tools to get started with writing automation for Fluid applications against a service.  It will focus on interactions with the service rather than automation in general, and will not cover the automation tools themselves or scenarios that do not require a service.
+
+{{% callout important %}}
+
+Specific examples of third-party libraries are used throughout this document for certain tasks.  While these are the same libraries used internally by Fluid, you are free to accomplish these tasks in other ways.  Microsoft is not responsible for [liability statement?].
+
+{{% /callout %}}
+
+## Automation against Tinylicious
+
+Automation against Tinylicious is useful for scenarios such as merge validation which want to be unaffected by service interruptions.  Your automation should be responsible for starting a local instance of Tinylicious along with terminating it once tests have completed.  One way you can do this is by using the `start-server-and-test` package in the following npm scripts.
```

Comment:
Could we include a link here for the `start-server-and-test` package and a short set of instructions to install it prior to running the scripts below?

Reply:
Will do.  Waiting on some guidance for how we want to talk about third-party code for how to format that